src/Components/WebAssembly/Build/testassets/blazorhosted/blazorhosted.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\standalone\standalone.csproj" />
+    <ProjectReference Include="..\razorclasslibrary\RazorClassLibrary.csproj" />
```

Comment:
Hmm, doing this invalidates quite a few other test cases that show transitively referenced static assets get published. Adding this here means it's no longer just a transitive reference.

Might it be worth adding a second RCL that is reference from both?

Reply:
Fair enough. I think we might not need this reference at all, since it is transitively brought in.